server/src/code-runner/docker-runner.ts

```diff
@@ -38,6 +40,30 @@ export class DockerRunner implements CodeRunner {
 
   private processCount = 0;
 
+  /**
+   * Run a command on a container conditionally.
+   *
+   * @param containerId ID of the container to run the command on.
+   * @param command Bash command to run.
+   * @param condition Condition which should evaluation to `true` in order to run the command.
+   */
+  private runCommand(containerId: string, command: string, condition = true) {
```

Comment:
I need to be able to run a command on the container conditionally. If someone knows a better or cleaner way. I also asked it on Twitter (https://twitter.com/SamVerschueren/status/1002035337194016768) and ended up with what I had. Feedback on this more than welcome!